.github/workflows/test-documentation.yml

```diff
@@ -31,16 +32,16 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-yarn-
 
-    # Select a branch on documentation to test against by adding text to the body of the
-    # pull request. For example: #documentation-branch: zoe-release-0.7.0
-    # The default is 'master'
-    - name: Get the appropriate documentation branch
+    # Select a branch on dapp to test against by adding text to the body of the
+    # pull request. For example: #dapp-encouragement-branch: zoe-release-0.7.0
+    # The default is 'main'
+    - name: Get the appropriate dapp branch
```

Comment:
The names in the body changed from `documentation` to 'dapp', but the file name is still 'test-documentation.yml`.

Reply:
Yes, it's testing the documentation using the dapp template which uses the term dapp generically